frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -63,23 +68,23 @@ export default class AccountsApi extends ExplorerApi {
 
   async getAccounts(
     limit: number = 15,
-    endTimestamp?: number
+    paginationIndexer?: AccountPagination
   ): Promise<AccountBasicInfo[]> {
     try {
       return await this.call("select", [
         `SELECT account_id as id, created_at_block_timestamp as createdAtBlockTimestamp, 
           created_by_transaction_hash as createdByTransactionHash, account_index as accountIndex
           FROM accounts
           ${
-            endTimestamp
-              ? `WHERE (created_at_block_timestamp * 1000000 + account_index) < :endTimestamp`
+            paginationIndexer
+              ? `WHERE created_at_block_timestamp <= :endTimestamp AND account_index < :accountIndex`
```

Comment:
This is not going to work.

account_index is a number that is sequentially assigned starting from 0 for every block, so many accounts will have `account_index` === 0, and this filter will filter out all the accounts with `account_index` >= 0.

```sql
WHERE created_at_block_timestamp < :endTimestamp OR (created_at_block_timestamp = :endTimestamp AND account_index <= :accountIndex)
```

Looking into it closer I think we are messing with things here. We are filtering the old accounts (where the timestamp is smaller than the given) and then we order them from newer to older (higher timestamp to lower timestamp)

Reply:
I use the first one since the second is not correct. It will always fetch first 15 accounts and not fetching the  following